tests/frontend/views/test_databases.py

```diff
@@ -1,9 +1,24 @@
+# Standard library
+from unittest.mock import patch
+
 # Third-party
+import botocore
 import pytest
 from django.conf import settings
 from django.urls import reverse
 from rest_framework import status
 
+# Original botocore _make_api_call function
+orig = botocore.client.BaseClient._make_api_call
+
+
+# Mocked botocore _make_api_call function
+def mock_make_api_call(self, operation_name, kwarg):
+    if operation_name == "CreateLakeFormationOptIn":
```

Comment:
Is this because `moto` doesnt mock this call?

Reply:
Yes, and it's how [moto recommends](https://docs.getmoto.org/en/latest/docs/services/patching_other_services.html) mocking out calls it hasn't implemented yet

Reply:
TIL